### PR TITLE
Move to io.opentelemetry.semconv:opentelemetry-semconv

### PR DESCRIPTION
**Pull request by deki**
**Description:**

[io.opentelemetry:opentelemetry-semconv](https://search.maven.org/artifact/io.opentelemetry/opentelemetry-semconv) was moved to [io.opentelemetry.semconv:opentelemetry-semconv](https://search.maven.org/artifact/io.opentelemetry.semconv/opentelemetry-semconv) and the former is now deprecated as per https://github.com/open-telemetry/opentelemetry-java/pull/5786.

**Existing Issue(s):**

#1044 

**Testing:**

Unittest execution

**Documentation:**

**Outstanding items:**

### CODE REVIEW

**Review by kenfinnigan — APPROVED** [review]
Thanks @deki

File: maven-extension/src/main/java/io/opentelemetry/maven/handler/GoogleJibBuildHandler.java [hunk 1]
```diff
@@ -86,8 +86,8 @@ public void enrichSpan(SpanBuilder spanBuilder, ExecutionEvent executionEvent) {
     spanBuilder.setAttribute(
         MavenOtelSemanticAttributes.MAVEN_BUILD_CONTAINER_REGISTRY_URL,
         "https://" + registryHostname);
-    spanBuilder.setAttribute(SemanticAttributes.HTTP_URL, "https://" + registryHostname);
-    spanBuilder.setAttribute(SemanticAttributes.HTTP_METHOD, "POST");
+    spanBuilder.setAttribute(SemanticAttributes.URL_FULL, "https://" + registryHostname);
```
Comment (cyrille-leclerc) [inline maven-extension/src/main/java/io/opentelemetry/maven/handler/GoogleJibBuildHandler.java:89]: I propose to decouple and solve later the question of adopting the new convention and to implement it aligning with the OTel Java Auto Instr that added a config flag `OTEL_SEMCONV_STABILITY_OPT_IN` / `otel.semconv-stability.opt-in`.
I created a PR narrowed down to the change of lib:
* https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1059

**Review by cyrille-leclerc — CHANGES_REQUESTED** [review]
Thank you very much for your contribution.
I'm sorry but I propose a slightly different path forward:

I propose to decouple and solve later the question of adopting the new convention and to implement it aligning with the OTel Java Auto Instr that added a config flag `OTEL_SEMCONV_STABILITY_OPT_IN` / `otel.semconv-stability.opt-in`.
I created a PR narrowed down to the change of lib:
* https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1059

**Comment by cyrille-leclerc** [comment]
FYI https://github.com/cyrille-leclerc/opentelemetry-java-contrib/tree/support-stable-http-semconv

**Comment by laurit** [comment]
Closing in favor of https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1299

**Pull request closed without merging**